Assignment5/NQueen/Position.java

```diff
@@ -0,0 +1,9 @@
+ public class Position 
+ {
+  int row, col;
```

Comment:
For best practice you have to use getter and setter methods to access these fields.
These fields should be private so that no one can change its value directly.